benchmark/smallbank-bench

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env node
+/* eslint-disable require-jsdoc */
+
+const {ClientService} = require('../');
+const fs = require('fs');
+
+(async () => {
+  const [
+    properties,
+    numberOfAccount,
+    numberOfThread,
+    duration,
+    rampUpTime,
+    operations,
+  ] = parseArgv();
+  const clientService = new ClientService(properties);
+  const accounts = Array.from(Array(numberOfAccount).keys());
+  const durationMillis = duration * 1000;
+  const rampUpTimeMillis = rampUpTime * 1000;
+
+  const errorCount = new Uint32Array(new SharedArrayBuffer(1024));
+  const counter = new Uint32Array(new SharedArrayBuffer(1024));
+  const totalCounter = new BigUint64Array(new SharedArrayBuffer(1024));
+  const latencyTotal = new BigUint64Array(new SharedArrayBuffer(1024));
+  let isRunning = true;
+
+  const start = Date.now();
+  let from = start;
+  for (let i = 0; i < numberOfThread; i++) {
```

Comment:
Is it using a single CPU core, right?
Java bench is using multiple cores using threads, so I'm wondering how we can compare them.

Reply:
@feeblefakie 

I think the async functions (the one that executes contracts) can leverage multiple CPU cores because they are non-blocking functions.
I am not exactly sure the details though.
Let me check this.

There is another way.
Node.js provides a concurrent practice called worker thread.
(It seems to me like a child process in runtime)
This for sure is leveraging multiple CPU cores.

If the `async function` practice doesn't really use multiple CPU cores,
then I will try to use worker thread to do it.